src/printer-python.js

```diff
@@ -0,0 +1,750 @@
+"use strict";
+
+const util = require("./util");
+
+const docBuilders = require("./doc-builders");
+const concat = docBuilders.concat;
+const join = docBuilders.join;
+const hardline = docBuilders.hardline;
+const line = docBuilders.line;
+const softline = docBuilders.softline;
+const group = docBuilders.group;
+const indent = docBuilders.indent;
+// const ifBreak = docBuilders.ifBreak;
+
+function printPythonString(raw, options) {
```

Comment:
Any way to better re-use some of `util.js printString()`?

Reply:
maybe, I think we need to make the js one a little bit more generic, in python you can have 2 different types of strings:
1. single quotes (eg. "hello")
2. triple quotes, for doc strings and multiline ("""hello world""")

All of them can have one or two modifiers (r, f and u). I'll think a bit on how to reduce code duplication :)